### PR TITLE
Update for Swift 5 compiler

### LINKED ISSUE #369: Xcode 10.2beta compilation issue
Hello,

There seems to be an issue when compiling Swifter on Xcode 10.2beta. The Errno.swift file produces a compilation error on [line](https://github.com/httpswift/swifter/blob/stable/Sources/Errno.swift#L13):

`return String(cString: UnsafePointer(strerror(errno)))`

The recommendation to fix this issue is to remove the call to the UnsafePointer. 

`return String(cString: strerror(errno))`

You can also follow the discusson on [apple forums](https://forums.developer.apple.com/thread/113919)

Cheers
Plato

### PR DESCRIPTION
**Pull request by alanzeino**
Resolves #369 and #367

### CODE REVIEW

File: Sources/HttpParser.swift [hunk 1]
```diff
@@ -33,9 +33,15 @@ public class HttpParser {
     }
     
     private func extractQueryParams(_ url: String) -> [(String, String)] {
+        #if compiler(>=5.0)
+        guard let questionMark = url.firstIndex(of: "?") else {
```
Comment (CoderXpert) [inline Sources/HttpParser.swift]: Should this be renamed to `questionMarkIndex`?

**Comment by Vkt0r** [comment]
Hey @alanzeino Thanks for taking the time to create this 👍

File: Sources/HttpParser.swift (hunk 1 above)
Comment (Vkt0r) [reply Sources/HttpParser.swift]: This was named before but it makes sense to rename IMO. @alanzeino Can you please update the name before merging it?

**Review by Vkt0r — APPROVED** [review]
Looks good!

**Pull request merged**